packages/apple-pay-integration/src/apple-pay-payment-strategy.ts

```diff
@@ -235,4 +253,27 @@ export default class ApplePayPaymentStrategy implements PaymentStrategy {
             );
         }
     }
+
+    private async _getDeviceData() {
```

Comment:
Can you move this function to a braintree related pakcage since it is obvious that this is the braintree way of collecting data?

Reply:
Did not understand a little bit. But we can't do it because the core logic is already placed in related package.

Reply:
I believe you should be able to move this function to a Braintree package and end up with:
```
getBraintreeDeviceData(loadPaymentMethod, braintreeIntegrationService): DeviceData
```